test/linter/test-consistency.ts

```diff
@@ -341,18 +368,29 @@ export class ConsistencyChecker {
 
       if (resolvedValue === null) {
         // We're not going to get a more specific version, so bail out now
-        return null;
+        continue;
```

Comment:
This change is causing a test failure. Specifically this test is failing after this change:
https://github.com/mdn/browser-compat-data/blob/0cfbcaa1d2b66f0a8637bdb4953daf29afb56fa1/test/linter/test-consistency.test.js#L32-L38

I think we might actually have two bugs here:
 - this should be a `continue`
 - code below should filter out all entries with `flags`

Reply:
I fixed this bug and got more linting errors. I'll file a separate PR for this too. Specifically:
```
Consistency - 3 problems (3 errors, 0 warnings):
 ✖ api.MediaMetadata - Error → Basic support in firefox was declared implemented in a later version (82) than the following sub-feature(s):
        → api.MediaMetadata.MediaMetadata: 71
        → api.MediaMetadata.album: 71
        → api.MediaMetadata.artist: 71
        → api.MediaMetadata.artwork: 71
        → api.MediaMetadata.title: 71
 ✖ api.Worker - Error → Basic support in nodejs was declared implemented in a later version (11.7.0) than the following sub-feature(s):
        → api.Worker.Worker: 10.5.0
        → api.Worker.message_event: 10.5.0
        → api.Worker.postMessage: 10.5.0
        → api.Worker.terminate: 10.5.0
 ✖ javascript.statements.export - Error → Basic support in nodejs was declared implemented in a later version (13.2.0) than the following sub-feature(s):
        → javascript.statements.export.namespace: 12.0.0
```

Reply:
I filed a split-out PR for this #16614.

Reply:
This concern was resolved when #16614 was merged into `main` and then into this branch.